repos/system_upgrade/el7toel8/actors/peseventsscanner/tests/unit_test.py

```diff
@@ -210,3 +221,21 @@ def consume_message_mocked(*models):
 
     with pytest.raises(StopActorExecutionError):
         scan_events('files/tests/sample02.json')
+    assert reporting.report_generic.called == 1
+    assert 'inhibitor' in reporting.report_generic.report_fields['flags']
+
+    with pytest.raises(StopActorExecutionError):
+        scan_events('files/tests/sample03.json')
+    assert reporting.report_generic.called == 2
```

Comment:
What do you think about cleaning "called" count, before checking for it again? Looking only for the test I become curious why first called is 1 and second is 2, since, in the end, the two calls for scan_events are not connected in fact.